tareas/clase-6/clase6-tarea1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Tarea 1 Clase 6</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="stylesheet" href="clase6-tarea1.css">
+    
+</head>
+<body>
+    <h1>Edades del grupo familiar</h1>
+    <form>
+        <label for="cantidad-integrantes">¿Cuántos integrantes hay en el grupo familiar?</label>
+        <input type="number" name="cantidad-integrantes" id="cantidad-integrantes" size="2" min="1" max="99">
+        <input type="button" id="boton-cantidad-integrantes" value="Enviar">
+    </form>
+    <form id="edades-integrantes">
+        <input type="button" class="oculto" name="boton-calcular-edades" id="boton-calcular-edades" value="Calcular">
+    </form>
+    <div id="mostrar-resultados" class="oculto">
+        <p><em id="edad-promedio">La edad promedio es:</em></p>
+        <p><em id="edad-menor">La menor edad es:</em></p>
+        <p><em id="edad-mayor">La mayor edad es:</em></p>
```

Comment:
Cuidado con el uso de etiquetas, son importantes para los search engine como Google, cualquier estilo que quieras darle a tu HTML, hacelo con CSS, `<em>` es para enfatizar algo en tu texto, te dejo un par de artículos por si te interesan

https://www.socialmediatoday.com/news/8-of-the-most-important-html-tags-for-seo/574987/
https://developer.mozilla.org/es/docs/Web/HTML/Element/em
https://www.educative.io/answers/html-tags-for-search-engine-optimization-seo